File: src/main/webapp/js/app/models/animal.js
App.Animal = DS.Model.extend({
    name: DS.attr('string'),
    age: DS.attr(),
    species: DS.attr('string'),
    breed: DS.attr('string'),
    drugs: DS.attr('string'),
    created: DS.attr('isodate'),
    sex: DS.attr('string'),
    status: DS.attr(),
    sterilized: DS.attr('boolean'),
    picture: DS.attr('string'),
    observations: DS.attr('string'),
    admission: DS.attr('isodate'),
    furPattern: DS.attr('string'),
    weight: DS.attr(),
    proportion: DS.attr(),
    primaryColor: DS.attr(),
    secondaryColor: DS.attr(),
    organization : DS.attr(),
//    organization: DS.belongsTo('organization'),

    isSmall: function () {
        return this.get('proportion') === 1;
    }.property('proportion'),

    isMedium: function () {
        return this.get('proportion') === 2;
    }.property('proportion'),

    isLarge: function () {
        return this.get('proportion') === 3;
    }.property('proportion'),

    bgimg: function () {
        return ('background-image: url("' + this.get("picture") + '");').htmlSafe();;
    }.property('picture')
});

File: src/main/java/eu/lpinto/petshelter/api/dto/Animal.java
package eu.lpinto.petshelter.api.dto;

import java.io.Serializable;
import java.util.Calendar;

/**
 *
 * @author Luis Pinto <code>- luis.pinto@tripdashboard.pt</code>
 */
public class Animal extends Entity implements Serializable {

    public static final long serialVersionUID = 1L;
    public Integer age;
    public String species;
    public String breed;
    public String drugs;
    public String sex;
    public Boolean sterilized;
    public Integer status;
    public String picture;
    public String observations;
    public Calendar admission;
    public String furPattern;
    public Integer proportion;
    public Float weight;
    public String primaryColor;
    public String secondaryColor;
    public Integer organization;

    public Animal() {
        super();
    }

    public Animal(final Integer id) {
        super(id);
    }

    public Animal(Integer age, final String species, final String breed, final String drugs, final String sex, final Boolean sterilized, final Integer status, final String picture, final String observations, final Calendar admission, final String furPattern, final Integer proportion, final Float weight, final String primaryColor, final String secondaryColor, final Integer organization, final Integer id, final String name, final Calendar created, final Calendar updated) {
        super(id, name, created, updated);
        this.age = age;
        this.species = species;
        this.breed = breed;
        this.drugs = drugs;
        this.sex = sex;
        this.sterilized = sterilized;
        this.status = status;
        this.picture = picture;
        this.observations = observations;
        this.admission = admission;
        this.furPattern = furPattern;
        this.proportion = proportion;
        this.weight = weight;
        this.primaryColor = primaryColor;
        this.secondaryColor = secondaryColor;
        this.organization = organization;
    }

}
